package.json

```diff
@@ -18,26 +18,18 @@
   },
   "description": "Safety-oriented, fp-first configuration of ESLint.",
   "devDependencies": {
-    "@babel/core": "^7.26.8",
-    "@babel/plugin-transform-typescript": "^7.26.8",
-    "@babel/preset-env": "^7.26.8",
     "@commitlint/cli": "^19.7.1",
     "@commitlint/config-conventional": "^19.7.1",
-    "@eslint/eslintrc": "^3.2.0",
```

Comment:
this one is unrelated to jest, but I forgot to remove this after switching to flat config